examples/light-switch-app/silabs/src/BindingHandler.cpp

```diff
@@ -85,6 +88,215 @@ void ProcessOnOffGroupBindingCommand(CommandId commandId, const EmberBindingTabl
     }
 }
 
+void ProcessLevelControlUnicastBindingCommand(BindingCommandData * data, const EmberBindingTableEntry & binding,
```

Comment:
static_cast on every argument is not the right way to do this. This argument types need to be set inside the structure as mentionned earlier.